homework/week-0/pooja/src/nyc/c4q/poojawins/PlayingCards.java

```diff
@@ -0,0 +1,31 @@
+package nyc.c4q.poojawins;
+
+/**
+ * Created by pooja on 5/1/15.
+ */
+
+import java.util.ArrayList;
+
+public class PlayingCards extends Game {
+    private ArrayList<Card> myDeck;
+
+    public PlayingCards() {
+        String[] suits = {"hearts", "spades", "diamonds", "clubs"};
+        String[] values = {"A", "2", "3", "4", "5", "6", "7", "8", "9", "10", "J", "Q", "K"};
+
+        myDeck = new ArrayList<Card>();
+
+        for (String suit : suits) {
```

Comment:
I love that the way you structured your code makes this nested loop easier to understand.